core/src/main/java/org/jruby/util/io/OpenFile.java

```diff
@@ -154,6 +154,9 @@ public static class Buffer {
 
     protected volatile Set<RubyThread> blockingThreads;
 
+    private Ptr spPtr = new Ptr();
+    private Ptr dpPtr = new Ptr();
```

Comment:
```suggestion
    private final Ptr spPtr = new Ptr();
    private final Ptr dpPtr = new Ptr();
```
... not sure how much (whether) it matters with C2 JIT these days